### PR TITLE
Webui-Aria2: Update request

### PR DESCRIPTION
**Pull request by Ansuel**
Outdated makefile that refer to old commit... need update for new change

### CODE REVIEW

**Comment by thess** [comment]
* PR needs maintainer notification and build/test confirmation (follow template if you don't know what I mean)
* PR and commit needs package prefix in comment - I don't know what package this is for.
Example: `webui-aria2: Update to 2016-10-03`
* Commit itself needs "Signed-off-By" tag

**Comment by Ansuel** [comment]
it's just a webui... i tested it and it works

File: net/webui-aria2/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=webui-aria2
-PKG_VERSION:=2016-05-30
+PKG_VERSION:=2016-10-3
```
Comment (yousong) [inline net/webui-aria2/Makefile]: `2016-10-3` works, but there is the convention: `2016-10-03`

**Comment by yousong** [comment]
> it's just a webui... i tested it and it works 

People write guidelines and prepare a template at issue/pr creation for a reason.  Please follow them when it applies.

Ping `webui-aria2` 's current maintainer @kuoruan

**Comment by kuoruan** [comment]
Please chenge the version to 2016-10-03 and add “Signed-off-By” to your commit.

**Comment by Ansuel** [comment]
Is it good now?

**Comment by kuoruan** [comment]
see https://github.com/openwrt/packages/blob/master/.github/pull_request_template
You can use ```git commit -s -m``` to sign off your commit.

**Comment by Ansuel** [comment]
I edit the code from the browser, i can't execute the command

**Comment by thess** [comment]
Corrected PR submitted by maintainer - closing this.

**Pull request closed without merging**